auth/src/lib.rs

```diff
@@ -38,17 +38,18 @@ pub async fn init(pool: PgPool, args: InitArgs, tier: AccountTier) -> io::Result
         None => ApiKey::generate(),
     };
 
-    query("INSERT INTO users (account_name, key, account_tier) VALUES ($1, $2, $3)")
-        .bind(&args.name)
+    query("INSERT INTO users (account_name, key, account_tier, user_id) VALUES ($1, $2, $3, $4)")
+        .bind("")
```

Comment:
Will a unique constraint fail if we end up inserting two machine accounts with no account_name? Maybe this should be `NULL` instead?

Reply:
There is no UNIQUE constraint on account_name remaining in my local db after the migrations, but it is still NOT NULL, hence this.